ezflow/data/dataset/base_dataset.py

```diff
@@ -101,6 +106,17 @@ def __getitem__(self, index):
         img1 = np.array(img1).astype(np.uint8)
         img2 = np.array(img2).astype(np.uint8)
 
+        img1 = torch.from_numpy(img1).permute(2, 0, 1).float()
+        img2 = torch.from_numpy(img2).permute(2, 0, 1).float()
+
+        img1, img2 = self.normalize(img1, img2)
```

Comment:
We can move the `normalize` operation to `line no: 152` and also add it` line no:131`

Reply:
Done